pkgs/modules/python/default.nix

```diff
@@ -3,7 +3,7 @@
 let
   pythonVersion = lib.versions.majorMinor python.version;
 
-  pkgs = if pythonVersion == "3.8" then pkgs-23_05 else pkgs-unstable;
+  pkgs = pkgs-unstable;
```

Comment:
If we're switching to pkgs-unstable, we can undo this change I was slightly complaining about here too and just reference pkgs directly: https://github.com/replit/nixmodules/pull/257/files#r1480135676